tests/unit_tests/test_build.py

```diff
@@ -184,8 +188,8 @@ def test_task_builds_with_valid_custom_resources(self, file):
                     "limit_memory": "3Gi",
                 }
 
-    def test_task_ignores_invalid_custom_resources(self, file):
```

Comment:
@gusrasch Is this test now removed?

Reply:
yes - it wasn't doing anything after the test case referenced in the commit from PR description

Reply:
got it.